rootfs/etc/pulse/client.conf

```diff
@@ -26,7 +26,7 @@ autospawn = no
 ; daemon-binary = /usr/bin/pulseaudio
 ; extra-arguments = --log-target=syslog
 
-; cookie-file =
+cookie-file = /run/pulse-cookie
```

Comment:
😆

Reply:
The default path doesn't exist ` ~/.config/pulse/cookie` and also if we create that, he have no the right to write into